README.md

```diff
@@ -268,13 +257,6 @@ Unless you [open a pull request](https://github.com/thoughtbot/neat/compare/), t
 - Opera 9.5+
 - Safari 5.1+
 
-## The Bourbon family
```

Comment:
Why remove this section? Neat is part of a larger family. Can we leave this as-is?